### PR TITLE
Search Results UI

### PR DESCRIPTION
**Pull request by Cleop**
Styles search results page #15

### CODE REVIEW

File: web/templates/search/results.html.eex [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+  <i class="fa fa-search white dib pr1-ns" aria-hidden="true"></i>
+  <%= form_for @conn, search_path(@conn, :new), [as: :search, method: :get, class: "dib"], fn f -> %>
+    <%= text_input f, :search, placeholder: "Enter search here", class: "bep-white-ph bep-bg-light-green white pa2 w-90 w5-ns ba br2 b--black-20 dib",  autofocus: true %>
+  <%= end %>
+  <%= link "Done", to: search_path(@conn, :index), class: "white dib pl1-ns" %>
+</section>
+<section class="ph2 w-50-ns center">
+  <h2 class="f6 pb2 bw1 bb bep-b--blue"> <%= @data["total"] %> Results</h2>
+  <ul class="ph1 list">
+    <%= for {evidence, i} <- Enum.with_index(@data["documents"]) do %>
```
Comment (SimonLab) [inline web/templates/search/results.html.eex]: :+1:

File: web/static/css/app.css [hunk 2]
```diff
@@ -51,3 +55,20 @@ body {
...
+color:#fff;
+}
+
+.bep-white-ph::-moz-placeholder { /* Firefox 19+ */
+color:#fff;
+}
+
+.bep-white-ph:-ms-input-placeholder {
+color:#fff;
+}
```
Comment (shouston3) [inline web/static/css/app.css]: With css like this, consider DRYing it:
```css
.bep-white-ph::-webkit-input-placeholder,
.bep-white-ph:-moz-placeholder, /* Firefox 18- */
.bep-white-ph::-moz-placeholder, /* Firefox 19+ */
.bep-white-ph:-ms-input-placeholder {
  color:#fff;
}
```

File: web/templates/search/results.html.eex [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
-<ul>
-  <%= for evidence <- @data["documents"] do %>
-    <li><%= link evidence["title"], to: evidence["link"] %> </li>
-  <% end %>
-</ul>
+<section class="bep-bg-green pv3 tc bb bep-b--blue bw1">
+  <i class="fa fa-search white dib pr1-ns" aria-hidden="true"></i>
+  <%= form_for @conn, search_path(@conn, :new), [as: :search, method: :get, class: "dib"], fn f -> %>
+    <%= text_input f, :search, placeholder: "Enter search here", class: "bep-white-ph bep-bg-light-green white pa2 w-90 w5-ns ba br2 b--black-20 dib",  autofocus: true %>
+  <%= end %>
```
Comment (shouston3) [inline web/templates/search/results.html.eex]: `<%= end %>` -> `<% end %>`

**Comment by SimonLab** [comment]
@shouston3 I've updated the PR based on your comments, I think it's ready for review/merge

**Pull request merged**